src/benchmark.zig

```diff
@@ -247,63 +247,61 @@ const TimedQueue = struct {
 
     pub fn lap(user_data: u128, operation: Operation, results: ClientError![]const u8) void {
         const now = std.time.milliTimestamp();
-        if (results) |value| {
-            if (log_level == .debug) {
-                const response = std.mem.bytesAsSlice(CreateAccountsResult, value);
-                log.debug("response={o}", .{response});
-            }
-
-            const self: *TimedQueue = @intToPtr(*TimedQueue, @intCast(usize, user_data));
-            const completed_batch: ?Batch = self.batches.pop();
-            assert(completed_batch != null);
-            assert(completed_batch.?.operation == operation);
-
-            log.debug("completed batch operation={} start={}", .{
-                completed_batch.?.operation,
-                self.batch_start,
-            });
-            const latency = now - self.batch_start.?;
-            switch (operation) {
-                .create_accounts => {},
-                .create_transfers => {
-                    if (latency > self.max_transfers_latency) {
-                        self.max_transfers_latency = latency;
-                    }
-                },
-                .commit_transfers => {
-                    if (latency > self.max_commits_latency) {
-                        self.max_commits_latency = latency;
-                    }
-                },
-                else => unreachable,
-            }
-
-            var batch: ?Batch = self.batches.peek();
+        const value = results catch |err| {
+            log.emerg("Client returned error={o}", .{@errorName(err)});
```

Comment:
What does the `{o}` specifier do? Should we use `{s}` for string here?

Reply:
you're right, `{s}` is the right choice.